app/views/episode_player/_form.html.erb

```diff
@@ -1,27 +1,39 @@
-<div class="row my-4 mx-2" data-controller="embed-preview"
-  data-embed-preview-embed-url-value="<%= embed_player_episode_url(episode, player_options, true) %>">
+<div class="row my-4 mx-2" data-controller="dynamic-form">
   <div class="col-8">
+    <a hidden href="<%= request.fullpath %>" data-dynamic-form-target="link"></a>
 
     <section class="container-fluid mb-5">
       <header>
         <h3 class="fw-bold"><%= t(".title.preview") %></h3>
       </header>
 
-      <%= render "form_preview", player_options: @player_options %>
+      <%= render "form_preview", embed_html: embed_player_episode_iframe(episode, player_options, true) %>
```

Comment:
And here's the trick - the `form_preview`s pass true at the end of this method, indicating they're _previewing_.  The below copyable form fields are all using the final non-preview urls.